app/services/testdata/TestDataGeneratorService.scala

```diff
@@ -215,13 +220,14 @@ trait TestDataGeneratorService extends MongoDbConnection {
   private def runInParallel[D <: CreateTestData, R <: CreateTestDataResponse](parNumbers: ParRange,
                                                                               createData: (Int => D),
                                                                               block: (Int, D) => Future[R])
-  : List[R] = {
-    parNumbers.map { candidateGenerationId =>
-      Await.result(
-        block(candidateGenerationId, createData(candidateGenerationId)),
-        10 seconds
-      )
-    }.toList
+  : List[R] = {parNumbers.map { candidateGenerationId =>
+        Await.result(
+          block(candidateGenerationId, createData(candidateGenerationId)),
+          30 seconds
```

Comment:
Intentional increment? Seems odd one extra user tripled this time. 🤔

Reply:
👍